ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml

```diff
@@ -67,3 +67,12 @@ spec:
         - mirrors:
           - registry.example.com:5000
           source: registry.redhat.io
+    #
+    # These CRs are to enable crun on master and worker nodes for 4.13+ only
+    #
+    # It's strongly recommended to include these CRs as day-0 extra manifests to
+    # avoid an extra reboot.
+    # - fileName: optional-extra-manifest/enable-crun-master.yaml
+    #   policyName: "config-policy"
+    # - fileName: optional-extra-manifest/enable-crun-worker.yaml
+    #   policyName: "config-policy"
```

Comment:
Because this is the reference configuration for 4.13 I would uncomment these and update the note above to say that for 4.12 and prior these lines should not be included.